torch/testing/_internal/mps_utils.py

```diff
@@ -0,0 +1,1012 @@
+            "nn.functional.multilabel_margin_loss",
+            "nn.functional.pdist",
+            "nn.functional.rrelu",
+            "nn.functional.norm",
+            "ormqr",
+            "pca_lowrank",
+            "qr",
+            "rsub",
+            "segment_reduce",
+            "_segment.reduce",
+            "segment.reduce",
+            "segment_reduce_offsets",
+            "_segment_reduce",
+            "_segment_reducelengths",
+            "_segment_reduceoffsets",
+            "sparse.mm",
+            "sparse.mmreduce",
+            "special.airy_ai",
+            "special.erfcx",
+            "special.hermite_polynomial_he",
+            "special.laguerre_polynomial_l",
+            "special.log_ndtr",
+            "special.ndtri",
+            "svd_lowrank",
+            "symeig",
+            "take",
+            "to",
+            "to_sparse",
+            "unique",
+            "vdot",
+            "segment_reduce_",
+            "geometric",
+            "geometric_",
+            "log_normal_",
+            "log_normal",
+            "cdouble",
+            "double",
+            "float_power",
+            "full_like",
+            "nonzero_static",
+            "nn.functional.adaptive_avg_pool1d",
+            "nn.functional.adaptive_avg_pool2d",
+        ]
+    },
+    **{
+        "linalg.matrix_norm": MPSSkipInfo(UNIMPLEMENTED, dtypes=[torch.float32]),
+        "linalg.norm": MPSSkipInfo(UNIMPLEMENTED, dtypes=[torch.float32]),
+        "linalg.normsubgradients_at_zero": MPSSkipInfo(
+            UNIMPLEMENTED, dtypes=[torch.float32]
+        ),
+        "norm": MPSSkipInfo(UNIMPLEMENTED, variant="nuc"),
+        "nn.functional.interpolate": [
+            MPSSkipInfo(UNIMPLEMENTED, variant="area"),
+            MPSSkipInfo(UNIMPLEMENTED, dtypes=[torch.uint8], variant="bicubic"),
+            MPSSkipInfo(UNIMPLEMENTED, variant="trilinear"),
+        ],
+        "scatter_reduce": [
+            MPSSkipInfo(
+                UNIMPLEMENTED,
+                dtypes=[torch.int64],
+                lower=15.0,
+                variant="amax",
+            ),
+            MPSSkipInfo(
+                UNIMPLEMENTED,
+                dtypes=[torch.int32, torch.int64],
+                upper=15.0,
+                variant="amax",
+            ),
+            MPSSkipInfo(
+                UNIMPLEMENTED,
+                dtypes=[torch.int64],
+                lower=15.0,
+                variant="amin",
+            ),
+            MPSSkipInfo(
+                UNIMPLEMENTED,
+                dtypes=[torch.int32, torch.int64],
+                upper=15.0,
+                variant="amin",
+            ),
+        ],
+        "special.chebyshev_polynomial_t": MPSSkipInfo(
+            UNIMPLEMENTED,
+            dtypes=[
+                torch.int16,
+                torch.bool,
+                torch.int32,
+                torch.int64,
+                torch.uint8,
+                torch.int8,
+            ],
```

Comment:
Nit (though it probably should have been the case for PR that added those
```suggestion
            dtypes=integral_types_and(torch.bool),
            ],
```